tensorboard/backend/security_validator.py

```diff
@@ -125,29 +122,32 @@ def _validate_csp_policies(self, policies):
 
     for directive in policies:
       name = directive.name
-      value = directive.value
-      has_default_src = has_default_src or name == _CSP_DEFAULT_SRC
+      for directive in directive.value:
```

Comment:
Sorry to ask for another change, but this should really still be called `value` or perhaps `source` (since it's a "source list") rather than `directive` since in the spec `directive` refers to the name and value together (basically the entire thing between semicolons) rather than only to the values like `'self'`.

Reply:
Changed it to `value` since future me would find `for source in directive.value` odd. 

Thanks for the feedback :D